src/main/drivers/serial_usb_vcp.c

```diff
@@ -40,8 +40,12 @@
 #include "usbd_hid_cdc_wrapper.h"
 #endif
 #include "usb_io.h"
-#elif defined(STM32F7)
+#elif defined(STM32F7) || defined(STM32H7)
```

Comment:
This can be simplified into
```
#elif defined(STM32F7)
#include "vcp_hal/usbd_cdc_interface.h"
#elif defined(STM32H7)
#include "vcph7/usbd_cdc_interface.h"
#endif
```

Reply:
Nah, the `#elif` section under question covers all the way to `#else` at the 5th line back from the final `#endif`.